### ISSUE TITLE
Build error - ccmap.vg not found

### ISSUE THREAD

**Issue by AndreasPK** [mpickering/eventlog2html#135]
On windows using ghc-8.10.2

```
Failed to build eventlog2html-0.8.0.
Build log (
C:\Users\Andi\AppData\Roaming\cabal\logs\ghc-8.10.2\eventlog2html-0.8.0-920ea66fa41e6027e4cab2d4dace3b6f6054c5c0.log
):
Preprocessing library for eventlog2html-0.8.0..
Building library for eventlog2html-0.8.0..
[ 1 of 16] Compiling Eventlog.Args    ( src\Eventlog\Args.hs, dist\build\Eventlog\Args.o )

src\Eventlog\Args.hs:15:1: warning: [-Wunused-imports]
    The import of ‘Data.Semigroup’ is redundant
      except perhaps to import instances from ‘Data.Semigroup’
    To import instances alone, use: import Data.Semigroup()
   |
15 | import Data.Semigroup ((<>))
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src\Eventlog\Args.hs:16:1: warning: [-Wunused-imports]
    The import of ‘Control.Applicative’ is redundant
      except perhaps to import instances from ‘Control.Applicative’
    To import instances alone, use: import Control.Applicative()
   |
16 | import Control.Applicative (optional)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[ 2 of 16] Compiling Eventlog.AssetVersions ( src\Eventlog\AssetVersions.hs, dist\build\Eventlog\AssetVersions.o )
[ 3 of 16] Compiling Eventlog.Javascript ( src\Eventlog\Javascript.hs, dist\build\Eventlog\Javascript.o )

src\Eventlog\Javascript.hs:54:23: error:
    • Exception when trying to run compile-time code:
        javascript/ccmap.vg: openBinaryFile: does not exist (No such file or directory)
      Code: embedFile "javascript/ccmap.vg"
    • In the untyped splice: $(embedFile "javascript/ccmap.vg")
   |
54 | treevega = decodeUtf8 $(embedFile "javascript/ccmap.vg")
   |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabal.exe: Failed to build eventlog2html-0.8.0 (which is required by
exe:eventlog2html from eventlog2html-0.8.0). See the build log above for
details.

```

**Comment by AndreasPK** [comment]
fwiw I was able to build from source when cloning the repo.

**Comment by AndreasPK** [comment]
It seems the file is simple not present on the hackage source tarball:

https://hackage.haskell.org/package/eventlog2html-0.8.0/src/

**Comment by mpickering** [comment]
I need to add this file to the cabal file.

**Comment by mpickering** [comment]
Fixed in 0.8.1
